src/azure-cli/azure/cli/command_modules/resource/_utils.py

```diff
@@ -76,3 +76,222 @@ def split_resource_id(resource_id):
         return subscription_scope, relative_resource_id
 
     return "/", relative_resource_id
+
+
+def json_min(js, **kwargs):
```

Comment:
I would suggest splitting `jsmin`-related functions to a separate file for easy management.

Reply:
OK~